src/sql/src/plan/query.rs

```diff
@@ -3104,6 +3103,21 @@ fn plan_list_subquery(
     .into())
 }
 
+fn plan_collate(
+    ecx: &ExprContext,
+    expr: &Expr<Aug>,
+    collation: &UnresolvedObjectName,
+) -> Result<CoercibleScalarExpr, anyhow::Error> {
+    if collation.0.len() == 2
+        && collation.0[0] == Ident::new("pg_catalog")
+        && collation.0[1] == Ident::new("default")
+    {
```

Comment:
there might be some better rustacian way to express this. If there is, I am not aware of it.